packages/design-system/vue/package.json

```diff
@@ -15,10 +15,15 @@
   "type": "module",
   "main": "./dist/index.js",
   "exports": {
-    "import": "./dist/index.js",
-    "require": "./dist/index.cjs"
+    ".": {
+      "import": "./dist/index.js",
+      "require": "./dist/index.cjs"
+    },
+    "./style.css": "./src/assets/style.css"
   },
   "dependencies": {
+    "@inkline/inkline": "^3.2.0",
```

Comment:
This should also have design-system-tokens as a dependency, right?

Reply:
It should.